modules/xml/src/main/resources/META-INF/mule-xml.xsd

```diff
@@ -514,14 +514,25 @@ The Jaxen filter allows you to route messages based on XPath expressions. The Ja
                     </xsd:element>
                     <xsd:element name="context-property" type="spring:entryType" minOccurs="0" maxOccurs="unbounded">
                         <xsd:annotation>
-                            <xsd:documentation>A property that wil be made available to the transform context. Expression Evaluators can be used to grab these properties from the message at runtime.
+                            <xsd:documentation>A property that will be made available to the transform context. Expression Evaluators can be used to grab these properties from the message at runtime.
                             </xsd:documentation>
                         </xsd:annotation>
                     </xsd:element>
                 </xsd:sequence>
+                <xsd:attribute name="timeBetweenTransformerEvictions" type="mule:substitutableInt">
+                    <xsd:annotation>
+                        <xsd:documentation>
+                          Transformers are pooled for better throughput, since performing an XSL
```

Comment:
The pool is prepopulated with the transformers so they are not constructed during message processing, but the presence of the pool does not affect the time of the actual transformation. What is the relationship of that with eviction?